xlators/mgmt/glusterd/src/glusterd-pmap.c

```diff
@@ -260,6 +261,7 @@ port_brick_bind(xlator_t *this, int port, char *brickname, void *xprt,
         if (ret > 0) {
             ret = 0;
             tmp_port->brickname = gf_strdup(new_brickname);
```

Comment:
nit: we probably could have released previous new_brickname and gf_asprintf() to it directly, instead of using new_brickname?

Reply:
Ack.
I thought this will look cleaner, though at the cost of more operations.